capivaraProjects/App.tsx

```diff
@@ -1,29 +1,16 @@
-/**
- * Sample React Native App
- * https://github.com/facebook/react-native
- *
- * Generated with the TypeScript template
- * https://github.com/react-native-community/react-native-template-typescript
- *
- * @format
- */
 import React from 'react';
 import {
   StyleSheet,
-  Text,
   View,
 } from 'react-native';
 import {Colors} from 'react-native/Libraries/NewAppScreen';
 import  Login from './src/views/Login';
-
-
+import {H1} from './src/components/H1';
 
 const App = () => {
   return (
       <View style={styles.container}>
-        <View style={styles.sectionViewTitle}>
-          <Text style={styles.sectionTextTitle}> Bem-vindo(a) à Taqtile!</Text>
-        </View>
+        <H1 title={'Bem-vindo(a) à Taqtile!'} />
```

Comment:
Porque não usou o children? 🤔 
```suggestion
        <H1>Bem-vindo(a) à Taqtile!</H1>
```